Undefined symbol GULResetLogger

Today's GoogleUtilities 5.4.0 release inadvertently introduced a breaking change.

A workaround is adding the following to the `Podfile`:

`  pod 'GoogleUtilities', '~> 5.3.7'`

```
Undefined symbols for architecture x86_64:
  "_GULResetLogger", referenced from:
      _FIRResetLogger in FirebaseCore(FIRLogger.o)
```

--- a/GoogleUtilities.podspec
+++ b/GoogleUtilities.podspec
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'GoogleUtilities'
-  s.version          = '5.4.0'
+  s.version          = '5.4.1'
   s.summary          = 'Google Utilities for iOS (plus community support for macOS and tvOS)'
 
   s.description      = <<-DESC

--- a/GoogleUtilities/CHANGELOG.md
+++ b/GoogleUtilities/CHANGELOG.md
@@ -1,5 +1,8 @@
 # Unreleased
 
+# 5.4.1
+- Fix GULResetLogger API breakage. (#2551)
+
 # 5.4.0
 - Update GULLogger to use os_log instead of asl_log on iOS 9 and later. (#2374, #2504)
 

--- a/GoogleUtilities/Logger/GULLogger.m
+++ b/GoogleUtilities/Logger/GULLogger.m
@@ -36,6 +36,12 @@ NSRegularExpression *GULMessageCodeRegex() {
   });
   return messageCodeRegex;
 }
+
+// GULResetLogger is unused but depended on by FirebaseCore versions 5.3.x and earlier.
+// It cannot be removed until GoogleUtilities does a breaking change release since FirebaseCore
+// allows GoogleUtilities to float up to the highest 5.x version.
+-void GULResetLogger() {
+}
 #endif
 
 @implementation GULLogger (Internal)

--- a/GoogleUtilities/Logger/GULLogger.m
+++ b/GoogleUtilities/Logger/GULLogger.m
@@ -40,7 +40,7 @@ NSRegularExpression *GULMessageCodeRegex() {
 // GULResetLogger is unused but depended on by FirebaseCore versions 5.3.x and earlier.
 // It cannot be removed until GoogleUtilities does a breaking change release since FirebaseCore
 // allows GoogleUtilities to float up to the highest 5.x version.
--void GULResetLogger() {
+void GULResetLogger() {
 }
 #endif
 

